utils/scripts/install-centos8.sh

```diff
@@ -1,17 +1,14 @@
-#!/bin/sh
+#!/bin/sh -e
```

Comment:
The -e and other options may be ignored on shebang.  Bash stops parsing the line at the first space.

Reply:
Interesting, but this won't be processed by bash, it's invoked by filename so the shebang line will be used and it'll be run under sh.

As this script only calls one command I think that the exit status of that command would be used as the exit status for the script anyway, so I'm not sure that the -e is actually required anyway.

Reply:
sh is just a link/symlink for bash on most distros.   Bash can look at the name it was invoked by and an implementation may disable bash extensions when the sh is used.   The shebang processing is not affected by that.

So the shebang options will still not be used.   Just last month, I had to clean up a couple of dozen scripts used by packer that were making the same assumption.

It would be better to get the shebang line cleaned up so that it is not used as the start of a script by someone else.

With a single command, the `set -e`  would not be needed.

Reply:
I know this is on a mac but the -e on the shebang line is honoured.
`
ashley@Ashleys-MacBook-Air ~ % ./mytest.sh
a
ashley@Ashleys-MacBook-Air ~ % cat mytest.sh
#!/bin/sh -e

echo a
false
echo b`

Reply:
Sorry, for the confusion, just checked with the bash source code and 1 parameter is permitted in the shebang line if bash handles the shebang.  If a syscall parses handles the shebang, then the bash code is not used.

I do not know why the parameters were ignored in all the shell scripts run by packer.